savoirfairelinux/sous-chef
# Fix internationalization support, now works in French

*Improves #60 and #119 by making French translations work.*

### Changes proposed in this pull request:

* Remove the en-ca locale folders (default is English anyway)
* Move the fr-ca translations to the "fr" locale folder and remove fr-ca folders.
  * AFAICT, fr-ca was broken. Now, anyone with fr (or fr_FR, fr_CA) in the Accept-Language header will see the translations.
* Update the documentation to remove references to fr-ca and en-ca and include procedure to generate .mo files.
* NOTE: no translations were lost, they were simply moved from "fr-ca" to "fr"

### How to test

In Chrome, go to chrome://settings/languages and make French (Canada) your topmost entry. See that you can see translations now (there aren't that many yet, but in the Clients list, the button to create a new Client should read "Nouveau client").

<img width="920" alt="screen shot 2016-08-27 at 9 46 59 pm" src="https://cloud.githubusercontent.com/assets/499162/18031120/d16400bc-6c9f-11e6-9218-0cb005191ae9.png">

### Status


Fill me in ...

Not sure why it was setup as en-ca and fr-ca but fr-ca simply wouldn't work for my setup which I'm guessing is a pretty common one. The new way is not restrictive to fr_CA but can include fr_FR, for example.

@savoirfairelinux/mll

See screenshot for proof that translations work (not a lot of them available!)
<img width="944" alt="screen shot 2016-08-27 at 9 44 55 pm" src="https://cloud.githubusercontent.com/assets/499162/18031119/b96b1d74-6c9f-11e6-8f8b-4f66bdf45deb.png">
